addons/explore-editor-theme/ExplorerDialog.tscn

```diff
@@ -45,3 +45,5 @@ margin_bottom = -4.0
 
 [node name="Constants" parent="VBoxContainer/TabContainer" instance=ExtResource( 4 )]
 visible = false
+
+[connection signal="filesystem_changed" from="VBoxContainer/TabContainer/Icons" to="." method="_notify_filesystem_change"]
```

Comment:
I prefer signal connections to be done explicitly in code rather than in the editor UI.